scripts/terraform/azure/postgresql/init-database.sql

```diff
@@ -0,0 +1,31 @@
+CREATE ROLE ctb_user WITH
```

Comment:
This file is to be run after the PG server has been created, right?

Reply:
Correct. There are multiple way of doing it, but as I said in my previous comment, I wanted to stick with native SQL language instead of coupling myself with vendor-specific post-database-creation workflows. This entire script could be more sophisticated, but since we're moving to AWS, I didn't spend too much time on the Azure one.